src/apps/replication/meta_server/server_state.cpp

```diff
@@ -503,6 +531,8 @@ void server_state::update_configuration(
             std::stringstream ss;
             ss << "/apps/" << app.app_name << "/" << old.gpid.pidx;
             partition_path = ss.str();
+            // TODO(qinzuoyan): last_drops is maintained only in meta_server and nerver used in replica_server,
+            // consider take it out of partition_configuration to avoid unnecessary data transmission.
```

Comment:
removed comment in https://github.com/qinzuoyan/rDSN/commit/eb135f225643b099bebcfd6a1f47239eb8d87d84